### ISSUE TITLE
Add license URL for photos

### ISSUE THREAD

**Issue by pstorch** [RailwayStations/RSAPI#26]
Mappings from license to URL can be found here RailwayStations/RSWebsite#20

**Comment by kaiwinter** [comment]
As I'm already into this... may I take this issue? Any suggestion where to put the Map for the mapping? In the [Station](https://github.com/RailwayStations/RSAPI/blob/master/src/main/java/org/railwaystations/api/model/Station.java) class maybe?

**Comment by kaiwinter** [comment]
[BaseStationLoader.createPhoto](https://github.com/RailwayStations/RSAPI/blob/master/src/main/java/org/railwaystations/api/loader/BaseStationLoader.java#L73) seems to be a better place. There is a country variable but it is the selected country not a passed client locale, right? If there will be a client locale in the future the BaseStationLoader could adjust the license link based on that country code.

**Comment by kaiwinter** [comment]
This is quite tricky as the station objects are created initially and are just returned from the cache on a request. Initializing all stations multiple times once for each possible language seems not as a good approach. I tried to adjust the link by a custom JsonSerializer but there is no access to the client language in the HttpHeaders from here.

So my current approach is to put the license URL in the station object(s) in `StationsResource.getWithCountry` and `StationsResource.getById` right before they are returned. I made a quick check with the VisualVM and couldn't see any performance impact. Note: As the license URL is not copied from the Photo object as the other variables, it is not included there.
